### PR TITLE
implementing InitialBox

### PR DESCRIPTION
**Pull request by akibria12**

### CODE REVIEW

File: src/components/posts/PostForm.js [hunk 1]
```diff
@@ -11,15 +12,7 @@ const PostForm = ({
...
-        <div className="profile-picture">
-          {loggedInUser && (
-            <img
-              src={`https://ui-avatars.com/api/?name=${loggedInUser.firstName}+${loggedInUser.lastName}&background=random&color=fff&rounded=true`}
-              alt="avatar"
-              height="50px"
-            ></img>
-          )}
-        </div>
+        <div className="profile-picture">{loggedInUser && <InitialsBox />}</div>
```
Comment (dogezen) [inline src/components/posts/PostForm.js:15]: does initialsBox need any PROPS?

File: src/components/posts/Post.js [hunk 2]
```diff
@@ -65,13 +66,9 @@ const Post = ({ post, loggedInUser, client, setPosts, posts, setErr }) => {
...
           <div className="single-post-author-img-container">
-            <div className="single-post-author-img">
-              <img
-                src={`https://ui-avatars.com/api/?name=${firstName}+${lastName}&background=random&color=fff&rounded=true`}
-                alt="avatar"
-                height="50px"
-              ></img>
-            </div>
+            {/* <div className="single-post-author-img"> */}
+            <InitialsBox />
```
Comment (dogezen) [inline src/components/posts/Post.js:70]: does initialsBox need any PROPS?

File: src/components/posts/Post.js [hunk 3]
```diff
@@ -65,13 +66,9 @@ const Post = ({ post, loggedInUser, client, setPosts, posts, setErr }) => {
...
+            {/* </div> */}
```
Comment (dogezen) [inline src/components/posts/Post.js:71]: remove commented out code

File: src/components/posts/CommentsList.js [hunk 4]
```diff
@@ -84,7 +87,9 @@ export default function CommentsList({ post, loggedInUser }) {
       )}
       <div className="add-new-comment-container">
         <div className="new-comment-user-img-container">
-          <div className="new-comment-user-img"></div>
+          <div className="new-comment-user-img">
+            <InitialsBox />
```
Comment (dogezen) [inline src/components/posts/CommentsList.js:91]: does initialsBox need any PROPS?

**Review by dogezen — DISMISSED** [review]

File: src/components/posts/PostForm.js (hunk 1 above)
Comment (akibria12) [reply src/components/posts/PostForm.js:15]: it does not need any props

File: src/components/posts/Post.js [hunk 5]
```diff
@@ -65,13 +66,9 @@ const Post = ({ post, loggedInUser, client, setPosts, posts, setErr }) => {
...
         <div className="single-post-header">
           <div className="single-post-author-img-container">
-            <div className="single-post-author-img">
-              <img
-                src={`https://ui-avatars.com/api/?name=${firstName}+${lastName}&background=random&color=fff&rounded=true`}
-                alt="avatar"
-                height="50px"
-              ></img>
-            </div>
+            {/* <div className="single-post-author-img"> */}
```
[the remark this answers is not in the document]
Comment (akibria12) [reply src/components/posts/Post.js]: it has been removed

**Pull request merged**